src/components/modules/epics/ContributionsEpicReminder.tsx

```diff
@@ -111,16 +110,22 @@ const isValidEmail = (email: string): boolean => {
     return re.test(String(email).toLowerCase());
 };
 
-interface Props extends ReminderFields {
+interface ContributionsEpicReminderProps {
+    reminderPeriod: string;
+    reminderLabel: string;
     // eslint-disable-next-line @typescript-eslint/ban-types
     onCloseReminderClick: Function;
 }
```

Comment:
This component doesn't need the `reminderCta` field so I think it's a little confusing passing in a prop that isn't used